ui/pages/swaps/index.js

```diff
@@ -374,11 +363,6 @@ export default function Swap() {
     return <></>;
   }
 
-  const isStxNotEnoughFundsError =
-    currentSmartTransactionsError === stxErrorTypes.NOT_ENOUGH_FUNDS;
```

Comment:
is this not an error we want to maintain?

Reply:
That's correct, we will fallback automatically to a regular swap if there is not enough funds for a gas fee via STX.